internal/cli/docs.go

```diff
@@ -199,9 +199,21 @@ description: "%s"
 				if len(f.Aliases) > 0 {
 					aliases := strings.Join(f.Aliases, "`, `-")
 
-					fmt.Fprintf(w, "- `-%s` (`-%s`) - %s\n", name, aliases, f.Usage)
+					str := fmt.Sprintf("- `-%s` (`-%s`) - %s", name, aliases, f.Usage)
+					if f.Default != "" {
+						str = fmt.Sprintf("%s The default is %s.", str, f.Default)
+					}
+					str += "\n"
+
+					fmt.Fprint(w, str)
 				} else {
-					fmt.Fprintf(w, "- `-%s` - %s\n", name, f.Usage)
+					str := fmt.Sprintf("- `-%s` - %s", name, f.Usage)
+					if f.Default != "" {
+						str = fmt.Sprintf("%s The default is %s.", str, f.Default)
+					}
+					str += "\n"
+
+					fmt.Fprint(w, str)
```

Comment:
Nit: It looks like this block and the one above on ~202 are identical, so we could move them outside of the `if-else` block. ex:

```go
var str string
if len(f.Aliases) > 0 {
  // alias stuff
  str = fmt.Sprintf(//things)
} else {
  str = fmt.Sprintf(//things)
}
if f.Default != "" {
	str = fmt.Sprintf("%s The default is %s.", str, f.Default)
}
str += "\n"
fmt.Fprint(w, str)
```

Reply:
Agreed! I think I beatcha by a few seconds. Got this simplified in [315ab8d](https://github.com/hashicorp/waypoint/pull/3501/commits/315ab8d5ccc858884ab8dd7375979cc1134182df)